index.html

```diff
@@ -1,6 +1,57 @@
-<head>
-  <script type="text/javascript" src="https://firefoxux.github.io/SharedBanner/fxux-banner.min.js"></script>
-</head>
-<body>
- <h1>Hello there!</h1>
-</body>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Firefox Design</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
```

Comment:
:P  I'ld _really_ rather we didn't use Tachyons for this page.  The repeated giant class strings are annoying, and making the CSS modular on a page of this size brings no benefits.